webapp/channels/src/utils/markdown/apply_markdown.ts

```diff
@@ -401,7 +401,9 @@ function applyBoldItalicMarkdown({selectionEnd, selectionStart, message, markdow
     };
 }
 
-export function applyLinkMarkdown({selectionEnd, selectionStart, message, url = 'url'}: ApplyLinkMarkdownOptions) {
+export const DEFAULT_PLACEHOLDER_URL = 'url';
```

Comment:
If introducing this const here, should we use this in other places in the function that reference `url`? like line 411

Reply:
The url in L:411 is a variable param not constant string. We are already doing it here

```
export function applyLinkMarkdown({selectionEnd, selectionStart, message, url = DEFAULT_PLACEHOLDER_URL}: ApplyLinkMarkdownOptions) {
```